### DISCUSSION TITLE
Add support for aria-label/aria-labelledby and role in ToastContainer

### DISCUSSION THREAD

**Discussion by fabianherrera15** [fkhadra/react-toastify#730] [Ideas]
First of all, thank you for this library and the effort put into making it accessible!
I'd like to see support for a `role` attribute that will be added to the ToastContainer element in conjunction with an `aria-label` or `aria-labelledby`, this will help screen reader users identifying this container as a landmark and facilitate navigation to it.
If implemented, the rendered element will look like this in the DOM for example`<div role="complementary" aria-label="notifications" class="Toastify__toast-container">`
The current workaround is to wrap the ToastContainer component with a `div` element that contains these attributes but ideally, we shouldn't bloat the DOM with more elements.

**Comment by fkhadra** [comment]
Hey @fabianherrera15 thanks for your feedback appreciate it. I'll consider adding that for the next release

**Comment by EGurney** [comment]
I would LOVE this.  It would make testing also a bit easier, as React Testing Library really pushes towards using labels and whatnot (for good reason).
